Source/santasyncservice/SNTSyncRuleDownload.m

```diff
@@ -92,12 +92,16 @@ - (BOOL)sync {
 
     uint32_t count = 0;
     for (NSDictionary *ruleDict in response[kRules]) {
+      self.syncState.rulesReceived++;
```

Comment:
Nit: Maybe consider moving the counting logic outside of the loop:
```
self.syncState.rulesReceived = response[kRules].count;
self.syncState.rulesProcessed = newRules.count;
```

(also, that `count` variable seems unnecessary)

Reply:
Done, sort of. count was unnecessary and rulesProcessed has been changed as suggested but because there are 2 loops here I need to add-assign rulesReceived.